site/content/docs/accounts-and-users/adding-private-locations.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Adding private locations
+weight: 51
+menu:
+  docs:
+    parent: "Accounts & Users"
+aliases:
+- "/docs/teams/adding-private-locations/"
+---
+
+**Checkly private locations is a technical preview feature. It is not recommended for production use and is available on request.**
+
+Note: While scheduled check runs work as expected, you cannot currently test checks from private locations. We are working on adding this soon. When saving an API check using the "Run & save" button, the test check will fail and you will need to click "Save this check anyway" to continue.
+
+Note: The Checkly Agent currently only supports x86/AMD64 architecture and not ARM64. This means Apple Mac M1 systems are not currently supported.
+
+## About Private Locations
+
+**You must have the Owner or Admin role on your Checkly account to create private locations.**
+
+Here are the requirements/questions before you get started:
+
+- Container runtime (we test using Docker, but other OCI-compliant runtimes should work)
+- Outbound internet access to https://agent.checklyhq.com (proxy configuration is supported)
+- Access to your internal API- or browser-based application
+- A Checkly account that you will use for testing Private Locations
+
+Here's how private locations work:
+
+A private location is similar to our existing public locations, but you have control over it. A private location can be in your own segregated network,  on-premises or cloud-based. You need one or more Checkly Agents installed in that location/network with access to your applications. Checkly checks are configured and scheduled in the Checkly cloud-based web UI as usual, but selecting a private location runs the check on the agent(s) in your infrastructure. Agents are stateless and ephemeral. As long as you have at least one operational agent, checks will run in your private location. Adding more agents will distribute the load and improve resilience automatically.
```

Comment:
Should we mention that there are currently differences in terms of chrome between public cloud and private locations?

Reply:
Let me make this comment irrelevant then. I created a PR so they use the same chromium binaries